x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -155,17 +216,18 @@ export const PolicyList = memo(() => {
         name: i18n.translate('xpack.securitySolution.policy.list.endpoints', {
           defaultMessage: 'Endpoints',
         }),
-        dataType: 'number',
         width: '8%',
         render: (policy: PolicyData) => {
+          const count = policyIdToEndpointCount.get(policy.id);
           return (
-            <PolicyEndpointListLink
+            <PolicyEndpointLink
               className="eui-textTruncate"
               data-test-subj="policyEndpointCountLink"
               policyId={policy.id}
+              nonLinkCondition={count === 0}
```

Comment:
For the first part where you are seeing the back to endpoint after you manually put in the url, that is expected behavior because `history.state` still contains the endpoint backlink information since you are using the same tab. If you were to put that policy details url into another tab, however, you should see back to policy list instead because the `history.state` is empty and fresh. Hope this makes sense.

Reply:
changing the `PolicyEndpointList` to be `PolicyEndpointCount` to clear up any confusion

Reply:
I see. Yes using the policy setting URL on a new tab shows the default back link.